sync/awsdatabricks.py

```diff
@@ -481,28 +481,39 @@ def _get_ebs_volumes_for_instances(
 ) -> List[dict]:
     """Get all ebs volumes associated with a list of instance reservations"""
 
+    def get_chunk(instance_ids: list, chunk_size: int) -> Iterator[list]:
```

Comment:
Isn't the return type technically a `-> Generator[list, None, None]`?

Might want to also type the args like so:

```suggestion
    def get_chunk(instance_ids: List[str], chunk_size: int) -> Generator[List[str], None, None]:
```

Reply:
Mentioning this because it can help the IDE auto-complete.

For example if you were to `[item.count('a') for item in self.get_chunk(***)` pycharm will autocomplete the `.count()` because it knows its a list of strings